Assets/Scenes/CopilotUI.unity

```diff
@@ -0,0 +1,4248 @@
+%YAML 1.1
```

Comment:
I'm unsure what this scene does. I opened it up and don't really see any copilot UI stuff in here

Reply:
The Copilots UI is turned of when opening the scene to simulate it being called/opened y the regular hub UI, You must turn the copilots object under the canvas on in order to see it working